src/components/CustomAppDetails/Versions.js

```diff
@@ -0,0 +1,271 @@
+import { useAlert, useConfig } from '@dhis2/app-runtime'
```

Comment:
This file has many different components and mixes a lot of business logic with presentation logic, from a code style and readability perspective I would like to see them separated out.